tests/token_integration_tests.rs

```diff
@@ -26,4 +27,55 @@ async fn test_token_operations() {
         1000,
     );
     assert!(transfer_ix.is_ok());
+}
+
+#[tokio::test]
+async fn test_token2022_operations() {
```

Comment:
It'd be better for these integration tests to use Kora and a known devnet t22 token like PYUSD to validate an e2e transfer operation works.